WooCommerce/Classes/ViewRelated/Orders/Order Creation/PaymentSection/GiftCardInputView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+/// Allows the user to enter a gift card code.
+struct GiftCardInputView: View {
+    @ObservedObject private var viewModel: GiftCardInputViewModel
```

Comment:
Should this be tagged as a `@StateObject` I don't think anything is retaining it, right?

Reply:
Yup I missed it, updated in https://github.com/woocommerce/woocommerce-ios/pull/10732/commits/1ee3cadb41b438648a838445154081d869057dbc